lib/streamlit/elements/widgets/time_widgets.py

```diff
@@ -429,7 +483,9 @@ def _time_input(
         )
 
         if widget_state.value_changed:
-            time_input_proto.value = serde.serialize(widget_state.value)
+            serialized_value = serde.serialize(widget_state.value)
+            if serialized_value is not None:
```

Comment:
nit: we can write this with the walrus operator now that our minimum python version is 3.8 (pretty inconsequential but it'd be nice if we could start using this pattern throughout the codebase)

```python3
if (serialized_value := serde.serialize(widget_state.value)) is not None:
    time_input_proto.value = serialized_value

Reply:
Oh, yes, indeed 👍  I wasn't aware that we can use walrus operators in our codebase